From fb88c75cb2475b08c2095ac165cd9c58d6f308a7 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Stefan=20D=C3=B6singer?= <stefan@codeweavers.com>
Date: Thu, 4 Jul 2013 21:00:17 +0200
Subject: [PATCH 069/127] wined3d: Don't force strict draw ordering for
 multithreaded CS
Reply-To: wine-devel <wine-devel@winehq.org>

Shouldn't be needed any more
---
 dlls/wined3d/wined3d_main.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dlls/wined3d/wined3d_main.c b/dlls/wined3d/wined3d_main.c
index 9abaac5..5edb922 100644
--- a/dlls/wined3d/wined3d_main.c
+++ b/dlls/wined3d/wined3d_main.c
@@ -311,8 +311,6 @@ static BOOL wined3d_dll_init(HINSTANCE hInstDLL)
         {
             TRACE("Enabling multithreaded command stream.\n");
             wined3d_settings.cs_multithreaded = TRUE;
-            TRACE("Enforcing strict draw ordering for multithreaded command stream.\n");
-            wined3d_settings.strict_draw_ordering = TRUE;
         }
     }
 
-- 
1.8.1.5

